### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by jacobvjk**
fixing a few typos

### CODE REVIEW

File: inst/extdata/PA2022CH_fr_exec_summary/scorecard.Rmd [hunk 1]
```diff
@@ -554,11 +554,11 @@ _Les informations pour cet indicateur n'ont pas été collectées dans le test c
...
+  * Mise en œuvre : conditions pour \guillemetleft OUI (type d'actif)\guillemetright , \guillemetleft NON\guillemetright , \guillemetleft PAS DE RÉPONSE\guillemetright définies comme ci-dessus mais pour des stratégies d'engagement. Pour les \guillemetleft autres actifs\guillemetright , les conditions ci-dessus doivent s'appliquer à au moins un des autres actifs. Pairs: Nombre d'utilisateurs ayant répondu OUI pour le type d'actif concerné sur l’ensemble des utilisateurs ayant investi dans ce type d'actif. Nombre d'utilisateurs ayant répondu OUI pour d'autres actifs sur le nombre d'utilisateurs ayant investi dans au moins un autre type d'actif. Les types d'actifs couverts sont "actions cotées (LE)", "obligations d'entreprises (CB)", "immobiliers (RE)", et "Autres actifs".
```
Comment (cjyetman) [inline inst/extdata/PA2022CH_fr_exec_summary/scorecard.Rmd:561]: is there supposed to be a space between open bracket and the text `\guillemetleft autres actifs\guillemetright`

File: inst/extdata/PA2022CH_fr_exec_summary/template.Rmd [hunk 2]
```diff
@@ -97,7 +97,7 @@ number_peers <- readr::read_csv(
...
+Ce résumé est une synthèse des résultats individuels de votre portefeuille téléchargé \startunderscoreletter {`r portfolio_name`}\stopunderscoreletter. \startunderscoreletter `r paste0(toupper(substr(investor_name, 1, 1)), substr(investor_name, 2, nchar(investor_name)))` \stopunderscoreletter a reçu un résumé par portefeuille. Un rapport interactif individuel détaillé (par portefeuille ou groupé) peut être consulté via un accès personnalisé sur le site platform.transitionmonitor.com en allemand, en français et partiellement en anglais. Un méta-rapport contenant des données anonymisées et agrégées de tous les participants, ainsi que des rapports sectoriels, est publié sur le site Internet de l'OFEV \textbf{(\href{https://www.bafu.admin.ch/bafu/fr/home/themes/climat/info-specialistes/climat-et-marche-financier.html}{lien})}.
```
Comment (cjyetman) [inline inst/extdata/PA2022CH_fr_exec_summary/template.Rmd:100]: should `platform.transitionmonitor.com` be linkified?

File: inst/extdata/PA2022CH_fr_exec_summary/template.Rmd [hunk 3]
```diff
@@ -235,7 +235,7 @@ tryCatch(
 
 ```{=latex}
 
-\* Technologies «à faible émission de CO\textsubscript{2}» (low-carbon): énergies renouvelables pour l'électricité et énergie électrique pour l'automobile. Technologies «à forte émission de carbone» (high-carbon): charbon, pétrole, gaz 
+** Technologies \guillemetleft à faible émission de CO\textsubscript{2}\guillemetright  (low-carbon): énergies renouvelables pour l'électricité et énergie électrique pour l'automobile. Technologies \guillemetleft à forte émission de carbone» (high-carbon): charbon, pétrole, gaz
```
Comment (cjyetman) [inline inst/extdata/PA2022CH_fr_exec_summary/template.Rmd:238]: last `\guillemetleft` has a space after it, and no apparent closing `\guillemetright`

File: inst/extdata/PA2022CH_fr_exec_summary/template.Rmd [hunk 4]
```diff
@@ -509,7 +509,7 @@ Ce graphique indique l'exposition actuelle aux technologies à faible émission
...
+Ce graphique affiche les scores d'alignement à la fois au niveau agrégé (barre la plus à gauche) et au niveau sectoriel. Le plus grand triangle bleu foncé indique le score obtenu par ce portefeuille et le plus petit triangle gris indique le résultat moyen obtenu par les pairs. La ligne horizontale délimite les scores alignés sur la trajectoire de 1,8 °C (ou plus) par rapport à ceux qui ne le sont pas. La trajectoire de 1,8°C correspond à l'objectif de l'Accord de Paris visant à limiter le réchauffement climatique à une hausse des températures \guillemetleft bien en dessous de 2°C\guillemetright. Les scores indiquent le scénario sur lequel s'alignent les plans de production des entreprises bénéficiant d’investissements. Pour obtenir de plus amples informations, veuillez vous reporter aux deux représentations ci-dessous.
```
Comment (cjyetman) [inline inst/extdata/PA2022CH_fr_exec_summary/template.Rmd:512]: space after `\guillemetleft`

File: inst/extdata/PA2022CH_fr_exec_summary/scorecard.Rmd (hunk 1 above)
Comment (jacobvjk) [reply inst/extdata/PA2022CH_fr_exec_summary/scorecard.Rmd:561]: no, but you mention a valid point, that I did not menage to fix yet. `\guillemetright` seems to automitically remove a space that follows. that means it is hard to use this well within a sentence. No problem if followed by a comma or a period. I have not figured out how to force a space in latex, which we would have to do in some cases

File: inst/extdata/PA2022CH_fr_exec_summary/template.Rmd (hunk 2 above)
Comment (jacobvjk) [reply inst/extdata/PA2022CH_fr_exec_summary/template.Rmd:100]: not according to the agreed upon templates

File: inst/extdata/PA2022CH_fr_exec_summary/template.Rmd (hunk 4 above)
Comment (cjyetman) [reply inst/extdata/PA2022CH_fr_exec_summary/template.Rmd:512]: not really sure if `\guillemetleft` needs to have a space after it or not?

Comment (jacobvjk) [reply inst/extdata/PA2022CH_fr_exec_summary/template.Rmd:512]: no space means it cannot tell where the command ends. the space seems needed and will be removed when rendered

Comment (cjyetman) [reply inst/extdata/PA2022CH_fr_exec_summary/template.Rmd:512]: possible that you can close it with `\guillemetleft\`?

**Comment by jacobvjk** [comment]
spacing is fixed

File: inst/extdata/PA2022CH_fr_exec_summary/template.Rmd (hunk 4 above)
Comment (jacobvjk) [reply inst/extdata/PA2022CH_fr_exec_summary/template.Rmd]: fixed spacing using ` \ ` where appropriate

File: inst/extdata/PA2022CH_fr_exec_summary/template.Rmd (hunk 3 above)
Comment (jacobvjk) [reply inst/extdata/PA2022CH_fr_exec_summary/template.Rmd]: spacing fixed

File: inst/extdata/PA2022CH_de_exec_summary/template.Rmd [hunk 5]
```diff
@@ -319,7 +319,7 @@ tryCatch(
...
+Der Aggregierte Score zeigt die Ausrichtung aller Vermögenswerte bis 2026 im Vergleich zum GECO 2021-Szenario. Der Score wird sowohl auf Portfolioebene (siehe auch Indikatoren gemäss "Swiss Climate Scores") als auch für jeden einzelnen PACTA-Sektor (ausser Zement, da nicht in GECO 2021 abgedeckt) berechnet.\footnote{Das GECO-Szenario wurde ausgewählt, da es die meisten PACTA-Sektoren abdeckt. Bitte beachten Sie, dass das GECO-Szenario grössere technologische Veränderungen erst nach dem Zeitraum von fünf Jahren annimmt, der in der PACTA-Analyse berücksichtigt wird. Daher kann ihr Aggregierter Score niedriger ausfallen, wenn ein anderes Szenario zur Berechnung benutzt wird.}
```
Comment (AlexAxthelm) [inline inst/extdata/PA2022CH_de_exec_summary/template.Rmd:322]: `{\ss}` -> ß, if that's preferred.

Comment (jacobvjk) [reply inst/extdata/PA2022CH_de_exec_summary/template.Rmd:322]: turns out the Swiss do not use it

Comment (AlexAxthelm) [reply inst/extdata/PA2022CH_de_exec_summary/template.Rmd:322]: I love that answer. 🇨🇭

**Review by AlexAxthelm — APPROVED** [review]

**Pull request merged**